Examples/green_button_data_example/parse_and_run_green_button.ipynb

```diff
@@ -0,0 +1,299 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d979ed0f",
+   "metadata": {},
+   "source": [
+    "This script takes Green Button Data XML (see https://greenbuttondata.org/index.html)\n",
+    "and transforms it into a format usable by SAM for bill savings calculations.\n",
+    "\n",
+    "Note that this was written based on data with a 15 minute timestep and at least one full calendar year of data\n",
+    "Future improvements to this script may allow for more flexible data inputs\n",
+    "\n",
+    "Also note: during testing we encountered data with PV generation where PV generation steps had a 0 for that timestep (when PV was exporting to the grid)\n",
+    "This script assumes load data does not include a generator that can export to the grid"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "79a8ae7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import certifi\n",
+    "import greenbutton_objects\n",
+    "import greenbutton_objects.parse\n",
+    "import json\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import PySAM\n",
+    "from PySAM import ResourceTools as tools\n",
+    "import PySAM.Battery as battery_model\n",
+    "import PySAM.Pvsamv1 as pv_model\n",
+    "import PySAM.Utilityrate5 as utility_rate\n",
+    "import PySAM.Cashloan as cashloan\n",
+    "import PySAM.UtilityRateTools\n",
+    "import requests"
```

Comment:
add `print(PySAM.__version__)` so we can show it's been tested with the latest version